package.json

```diff
@@ -13,7 +13,8 @@
     "dotenv": "^5.0.0",
     "express": "^4.16.2",
     "nodemailer": "^4.4.0",
-    "popsicle": "^9.2.0"
+    "popsicle": "^9.2.0",
```

Comment:
NB: Let's add an eslint rule to enforce trailing commas in order to preserve blame. I'll file an issue soon.

Reply:
Well _actually_... No trailing commas here, because it's a JSON file and I'm sure we'd break some stuff with invalid JSON markup in a package.json, but I see your point (re: #40).

Reply:
Oh no, you're right! Thanks! *shakes fist at the JSON gods*

Reply:
But _actually_, now that my eyes have seen this, I'll disagree with the use of _both_ **popsicle** _and_ **request** modules, since they both seem to roughly do the same thing. We should probably use one or the other (or [**isomorphic-fetch**](https://www.npmjs.com/package/isomorphic-fetch) or [**got**](https://www.npmjs.com/package/got) or ...)

Reply:
Yeah. I will file an issue to get rid of our popsicle usage.

Reply:
Ref: #42